src/CategoryCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardHeader, IconButton, Avatar } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(1),
+    maxWidth: 500,
```

Comment:
We should probably also set `minWidth` to make sure Cards are same size :) now, a card with smaller content will shrink:
![image](https://user-images.githubusercontent.com/8350925/106358603-84e11400-630d-11eb-991a-1587ed225293.png)

Reply:
I've tried to fix it. I believe it works, but I'm not sure, so let me know if there's anything I can adjust.

Reply:
Yes, the cards now take up all available space which is the behavior we want here :)